packages/core/src/components/Button/Button.spec.tsx

```diff
@@ -403,6 +403,68 @@ describe('Button', () => {
     });
   });
 
+  describe('Attention', () => {
+    primaryBtnSpecs.forEach((spec) =>
```

Comment:
This will test the Primary button, not the Attention button. If I'm not mistaken, it passes the "primary" variant to the Button component. 
And I see the same mistake in the Info button test.

Please fix this mistake for the Attention and the Info button

Reply:
Done